tests/data/python2.py

```diff
@@ -5,10 +5,11 @@
 print >> sys.stderr , "Warning:" ,
 print >> sys.stderr , "this is a blast from the past."
 print >> sys.stderr , "Look, a repr:", `sys`
+print >> sys.stderr , U"Even these prefixes are normalized!"
 
 
 def function((_globals, _locals)):
-    exec ur"print 'hi from exec!'" in _globals, _locals
+    exec "print 'hi from exec!'" in _globals, _locals
```

Comment:
Why are the `ur` removed here?

Reply:
Oh wow. "ru" is invalid but "ur" isn't? 😅 Good catch. Let me go through the others once more.

Reply:
"br" is valid and "rb" invalid. Well. This is a bummer. They go exactly against the rule. Should we go with "r last" after all?

Reply:
Oh you're right. And in Python 3 they're both invalid.

Reply:
Yeah "u" with anything else seems to be invalid, but "br" naturally works.

Reply:
Sorry that comment was in reference to `ru` not `rb`.

In summary:
- Python 3 allows `rb`, `br`, but not `ur` or `ru`
- Python 2 allows `br` and `ur` but not `rb` or `ru`

Reply:
I commented on the original issue and marked this PR as a draft. We ought to discuss it through before merging anything.